### DISCUSSION TITLE
New repo and/or branch for source of PyPI package

### DISCUSSION THREAD

**Discussion by castedo** [castedo/sshsig#1] [Ideas]
@jelmer I have this mirror of source code that currently gets packaged up into a top-level `sshsig0` module that gets packaged into https://pypi.org/project/hidos/ version 2.0.

I figure the source code for a new `sshsig` PyPI package that you maintain long-term should not be maintained from this `castedo/sshsig` mirror repo. Do you think it should be a new repo `jelmer/sshlib`?

My main interest is in deploying SSHSIG documentation that is independent of the Dulwich documentation. My hunch is having a repo that is separate from Dulwich will make having separate SSHSIG documentation easier.

**Comment by jelmer** [comment]
Let's just start with this repository? You can either give me access to it, or I can create PRs for the moment - if I do need to create a fork at some point I always can. 

But it would be good to have both of us listed as maintainers on pypi; my username there is also "jelmer".

**Reply by castedo** [reply]
OK, i'll proceed focusing on this repo `castedo/sshsig` and give you access to it. I need to change how mirroring is done between gitlab and github. Whatever I'll I end up doing, I will try and make the `main` branch here just a typical `main` branch that lives here in github.

I might play around with having a feature branch or special branch of something that is mirrored from gitlab. But I'll try and keep the `main` branch simple.

**Reply by castedo** [reply]
FYI, I've added metadata to `pyproject.toml` and listed both of us under the `maintainers` key.

https://github.com/castedo/sshsig/commit/0a906b941b861f452d598f3af2ea3d62cc355c05
